src/routes/reviewRoutes.js

```diff
@@ -7,11 +7,16 @@ const {
   updateReview,
   getAllReviews,
   getSingleReview,
+  getAllReviewsForThisProduct,
 } = require('../controllers/reviewController');
 
 router.route('/').get(getAllReviews);
 
+
 router.route('/album/:albumId').post(authenticateUser, createReview);
 
 router.route('/:id').patch(authenticateUser, updateReview).get(getSingleReview)
+
+router.route('/album/:albumId').get(getAllReviewsForThisProduct)
```

Comment:
This can just be tacked onto line 16:

`router.route('/album/:albumId').post(authenticateUser, createReview).get(getAllReviewsForThisProduct)`